src/fetchStoryHtml.ts

```diff
@@ -45,11 +45,14 @@ const fetchStoryHtml = async (
 
   const fetchUrl = new URL(`${url}/_cl_server`);
   const init: {
+    _componentFileName: string;
+    // Preserve backward compatibility with older versions of drupal/cl_server:
     _storyFileName: string;
     _drupalTheme: string;
     _variant?: string;
     _params: string;
   } = {
+    _componentFileName: context.parameters.fileName,
     _storyFileName: context.parameters.fileName,
```

Comment:
I updated this PR to include _both_ parameters, which preserves backward compatibility. See [accompanying commit](https://git.drupalcode.org/project/cl_server/-/merge_requests/19/diffs?commit_id=82573dd42a1580d22e43c0a5e430b3cbfc7ce0c4) for `cl_server`, which deprecates (instead of dropping) `_storyFileName`.

This should make for a smoother transition.